src/contexts/useSdkConfigurationService.tsx

```diff
@@ -0,0 +1,63 @@
+import { getSdkConfiguration } from '~utils/onfidoApi'
+import { useContext, useEffect, useState } from 'preact/compat'
+import { h, ComponentChildren, createContext, Fragment } from 'preact'
+import { SdkConfiguration } from '~types/api'
+import deepmerge from 'deepmerge'
+
+type SdkConfigurationServiceProviderProps = {
+  children: ComponentChildren
+  url?: string
+  token?: string
+  fallback?: ComponentChildren
+}
+
+const defaultConfiguration: SdkConfiguration = {}
+
+const SdkConfigurationServiceContext = createContext<SdkConfiguration>(
+  defaultConfiguration
+)
+
+export const SdkConfigurationServiceProvider = ({
+  children,
+  url,
+  token,
+  fallback,
+}: SdkConfigurationServiceProviderProps) => {
+  const [configuration, setConfiguration] = useState<
+    SdkConfiguration | undefined
+  >(undefined)
+
+  useEffect(() => {
+    if (!url || !token) {
+      return
+    }
+    getSdkConfiguration(url, token)
+      .then((configuration) => {
+        let mergedConfiguration = deepmerge(defaultConfiguration, configuration)
+
+        if (process.env.NODE_ENV === 'development') {
```

Comment:
Humm why?

Reply:
Oh no, I meant why only for `development`?

Reply:
* The default configuration will be defined in this file. It's the fallback configuration.
* In the webpackconf, it's only a way for devs to override a configuration value without touching the default value.

Reply:
> In the webpackconf, it's only a way for devs to override a configuration value without touching the default value.

Well I guess we can simplify the _default_ `SDK_CONFIGURATION` at the front-end. This default value will be the fallback in case we are unable to load the remote configuration. So in each env, we'll expect a different value of this. But for the default value, we'll make it ... default, like no MFC, etc.

So IMO we'll be fine to have `SDK_CONFIGURATION` for `PROD_CONFIG` and propagate it to other envs

Reply:
👍